tasks/web-install-py3.yml

```diff
@@ -120,7 +102,7 @@
       }}
     state: present
     virtualenv: "{{ omero_web_virtualenv_basedir }}"
-    virtualenv_command: /usr/bin/python3 -mvenv
+    virtualenv_command: /usr/local/bin/ome-python3-virtualenv.sh
```

Comment:
My preference would be to not include `.sh` in case it becomes `.py` or something in the future. Other than that, despite being a mouthful, `ome-python3-virtualenv` is nicely namespaces if we're going to need to add unexpected helper scripts.

Reply:
See https://github.com/manics/ansible-role-python3-virtualenv/commit/53dbc2b860f8e55bad4288ce459479e2a0e9abed

Reply:
Release as 0.1.0 or 1.0.0?

Reply:
`0.1.0`? Some of the other dependencies are still `<1` https://github.com/ome/ansible-role-omero-web/blob/3.0.1/meta/main.yml

Reply:
I'd think `0` versus `1` is less about what the version of the other dependencies and more a gauge of how comfortable you are with the API and its stability. But if that's `0.1.0` as well, WFM.

Reply:
Go with `0.1.0`, if there are no unexpected changes then go to `1.0.0` whenever there's a new update to the role?

Reply:
Tagged.

Reply:
Looks like it needs some travis magic https://travis-ci.org/ome/ansible-role-python3-virtualenv

Reply:
Fixed  - see https://galaxy.ansible.com/ome/python_virtualenv. Is the discrepancy between the GitHub repository name and the role name on purpose (`python` vs `python3`)?

Reply:
Oops, it's a typo. What the easiest way to resolve it, rename the role or rename the repo?